src/Traits/UseHooks.php

```diff
@@ -258,6 +262,51 @@ public function hookActionAdminControllerSetMedia(): void
         }
     }
 
+    /**
+     * This hook adds buttons for the module catalogue toolbar
+     *
+     * @param array $params
+     *
+     * @return array
+     */
+    public function hookActionAdminModuleExtraToolbarButton(array $params)
+    {
+        /** @var DataProvider $addonsProvider */
+        $addonsProvider = $this->get('mbo.addon.module.data_provider.addons');
```

Comment:
What is the difference between this and the same logic in [`ModuleCatalogController`](https://github.com/PrestaShopCorp/ps_mbo/pull/127/files#diff-351301f3adf9cb1fcc43b4a2673cc686e98b0a296ac03a439355ac9b1180aa24R224) ? Do we need both ? And if yes, can we factorize this portion of code ?